### ISSUE TITLE
Indented code in markdown cells

### ISSUE THREAD

**Issue by mbelletti** [HappenApps/Quiver#862]
How can I revert to the old behavior when inline code was rendered as code cell?

**Comment by ylian** [comment]
Can you show a screenshot? I'm not aware that the behavior has changed.

**Comment by mbelletti** [comment]
Sorry, I mean indented code not inline.

**Comment by ylian** [comment]
Yeah, I thought you meant indented code. But I'm not aware of behavior changes. Can you give me more details? Thanks.

**Comment by mbelletti** [comment]
Now indented code is rendered like this:
![now](https://cloud.githubusercontent.com/assets/493145/25620685/81821548-2f4f-11e7-90fe-8a4feb496fda.png)

but before it was rendered like this:
![before](https://cloud.githubusercontent.com/assets/493145/25620710/95571c58-2f4f-11e7-8691-f2b322601b92.png)

that is what I would :)

**Comment by ylian** [comment]
You might have changed your settings. Check Preferences > Cells > Code Cells > Show line numbers.

If it still doesn't work, let me know. Could be a theme issue as well.

This is what I have with the current version:

![image](https://cloud.githubusercontent.com/assets/984895/25620917/36998e84-2f50-11e7-8099-b3d5807a4b58.png)

**Comment by ylian** [comment]
Also maybe we use indented code blocks differently. This is what I use:

![image](https://cloud.githubusercontent.com/assets/984895/25621007/6b1bb86c-2f50-11e7-9a22-6a9538b12836.png)

**Comment by mbelletti** [comment]
I'm using 3.0.6 (99) from app store.

Ah! Using the triple quotes it works. 

But before wasn't' necessary. 

Is there a way to get back the old behavior? Or at least an option to handle indented code.
I've tons of docs wrote in that way...

**Comment by ylian** [comment]
How do you use indented code blocks? Using 4 spaces? I thought only fenced code blocks got translated to code cells, but I will double check with the old version. Anyway I will make it backward compatible. :)

**Comment by ylian** [comment]
A small example would be helpful. So I can be sure it will be fixed for you.

**Comment by mbelletti** [comment]
I'm using 4 spaces for indented code.
Take a look here [issues/478](https://github.com/HappenApps/Quiver/issues/478)

I expect that this raw text:
```
Ottengo la lista dei dischi

    diskutil list
```

is rendered this way:
![before](https://cloud.githubusercontent.com/assets/493145/25621790/aef97c84-2f52-11e7-9d05-bccbdbe84ac9.png)

Thank you

**Comment by ylian** [comment]
Yeah, that's a good reference to the old behavior. I will make it backward compatible.

Keep this open until the behavior is backward compatible.

**Comment by mbelletti** [comment]
Hi @ylian, any news on that?

**Comment by mbelletti** [comment]
up
